src/components/users/login/LoginPage.js

```diff
@@ -21,11 +21,26 @@ const LoginPage = () => {
       .then(res => {
         localStorage.setItem(process.env.REACT_APP_USER_TOKEN, res.data.data.token)
         setLoginResponse(res.data)
+        console.log(res.data)
         navigate("../posts", { replace: true });
       })
       .catch(err => console.log(err.response))
   }
 
+
+// function getListOfCohorts() {
```

Comment:
Should also remove this if it is not needed anymore

Reply:
Done.